velox/expression/CastExpr.cpp

```diff
@@ -14,14 +14,14 @@
  * limitations under the License.
  */
 
-#include "velox/expression/CastExpr.h"
```

Comment:
nit: by our code convention, header file of the same name as the cpp file is supposed to be included first with a blank line afterward that separates it from other headers. So the original code here is good I think.

Reply:
Understood. Updated, thanks.